src/SymfonyCache/HttpCacheAwareKernelInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\SymfonyCache;
+
+use Symfony\Component\HttpKernel\HttpCache\HttpCache;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+interface HttpCacheAwareKernelInterface extends HttpKernelInterface
+{
+    /**
+     * @param HttpCache $httpCache
+     */
+    public function setHttpCache(HttpCache $httpCache);
+
+    /**
+     * @return HttpCache
```

Comment:
do we need a HttpCache specifically? i think the client only cares about `handle`, so we could use `KernelInterface` as the thing (but name it $httpCache nonetheless)